### PR TITLE
[CRAW-290] Fix for the 'Page is not recrawled after repeating inject' bug.

### PR DESCRIPTION
**Pull request by vanchpuck**

### CODE REVIEW

File: src/java/org/apache/nutch/crawl/InjectorJob.java [hunk 1]
```diff
@@ -173,6 +173,8 @@ protected void map(LongWritable key, Text value, Context context)
       WebPage row = WebPage.newBuilder().build();
       row.setFetchTime(curTime);
       row.setFetchInterval(customInterval);
+      // Set the previous fetch time to recrawl page, if it was already crawled
+      row.setPrevFetchTime(0L);
```
Comment (SergeyWeiss) [inline src/java/org/apache/nutch/crawl/InjectorJob.java]: What if we implement ScoringFilter that would do just this?

Comment (kgusakov) [inline src/java/org/apache/nutch/crawl/InjectorJob.java]: +1

File: src/plugin/build.xml [hunk 2]
```diff
@@ -150,6 +152,7 @@
     <ant dir="protocol-sftp" target="clean"/>
     <ant dir="scoring-link" target="clean"/>
     <ant dir="scoring-opic" target="clean"/>
+  	<ant dir="scoring-recrawl-injected" target="clean"/>
```
Comment (kgusakov) [inline src/plugin/build.xml]: Indent?

File: src/plugin/scoring-recrawl-injected/ivy.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<ivy-module version="1.0">
+
+    <info organisation="org.integralads.crawler" module="${ant.project.name}">
```
Comment (kgusakov) [inline src/plugin/scoring-recrawl-injected/ivy.xml]: Not org, but com.

**Comment by SergeyWeiss** [comment]
:+1:

**Pull request merged**